UnityProject/Assets/Scripts/Core/Lighting/LightSource.cs

```diff
@@ -306,11 +307,20 @@ private void TryRemoveBulb(HandApply interaction)
 		{
 			try
 			{
-				var handSlot = interaction.PerformerPlayerScript.DynamicItemStorage.GetActiveHandSlot();
+				var handSlots = interaction.PerformerPlayerScript.DynamicItemStorage.GetNamedItemSlots(NamedSlot.hands);
 
-				if (mState == LightMountState.On && (handSlot.IsOccupied == false ||
-				                                     !Validations.HasItemTrait(handSlot.ItemObject,
-					                                     CommonTraits.Instance.BlackGloves)))
+				bool HasGlove()
+				{
+					foreach (var slot in handSlots)
+					{
+						if(slot.IsEmpty) continue;
+						if (Validations.HasItemTrait(slot.ItemObject, gloveTrait)) return true;
```

Comment:
Why did you stop using the common trait?

Reply:
something was wrong with unity on my end, it fixed itself so no need for this change.